gpu/kinfu/tools/kinfu_app.cpp

```diff
@@ -84,10 +84,10 @@ using ScopeTimeT = pcl::ScopeTime;
 
 #include "../src/internal.h"
 
-using namespace std;
 using namespace pcl;
 using namespace pcl::gpu;
 using namespace Eigen;
+using namespace std::chrono_literals;
```

Comment:
Where is this needed? Just interested to knnow

Reply:
Line 1010, there's an `ms` literal:

`        bool has_data = (data_ready_cond_.wait_for(lock, 100ms) == std::cv_status::no_timeout);`